_gsocprojects/2020/project_CMS.md

```diff
@@ -0,0 +1,8 @@
+---
+project: CMS
+layout: default
+logo: CMS-logo.png
+description: |
+  [CMS](http://cms.cern/) is a high-energy physics experiment at the [Large Hadron Collider](http://home.web    .cern.ch/topics/large-hadron-collider) (LHC) at [CERN](http://home.cern/).  It is a general-purpose detector that is designed to observe any new physics phenomena that the LHC might reveal. CMS acts as a giant, high-speed camera, taking 3D "photographs" of particle collisions from all directions up to 40 million times each second. The CMS collects few tens of Peta-Bytes of data each year and processes them through Worldwide LHC Computing Grid infrastructure around the globe. 
```

Comment:
There are still problems with formatting for this file, but I will fix separately to fasten the process.